vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -632,6 +649,17 @@ void AdsImpl::CheckReadyAdServe(const bool forced) {
       return;
     }
 
+    if (IsDoNotDisturb() && !IsForeground()) {
+      // TODO(Terry Mancey): Implement Log (#44)
```

Comment:
is this TODO still needed?

Reply:
This TODO is needed as it is part of another ticket to make sure logging is added based upon Muon